index.js

```diff
@@ -39,13 +39,14 @@ var hostedGitInfo = require('hosted-git-info');
  *
  */
 module.exports = function (target, dest, opts) {
-    var fetchArgs = opts.link ? ['link'] : ['install'];
     opts = opts || {};
+    var manager = opts.manager || 'npm';
+    var fetchArgs = opts.link ? ['link'] : manager === 'yarn' ? ['add'] : ['install'];
     var tree1;
     var nodeModulesDir = dest;
 
     // check if npm is installed
```

Comment:
comment is out of date

Reply:
What do you mean ?

Reply:
this line says that it checks if "npm is installed". this is not true any more, also pretty self explaining now per function name, so comment can just be removed.